twisted_tools/buildbot.py

```diff
@@ -11,15 +12,29 @@
 
 
 
-def getURLForBranch(branch):
```

Comment:
make it private so that I don't need to write specific tests for it and making sure people are not randomly using this method.